Failing action runs due to FileNotFoundException

While refactoring your `bach` repository, it seems like this action got broken:

```lang-none
Run sormuras/download-jdk@v1
Download JDK 14 (linux-x64)
Exception in thread "main" java.io.FileNotFoundException: https://github.com/sormuras/bach/raw/HEAD/install-jdk.properties
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1920)
	at java.base/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1520)
	at java.base/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:250)
	at java.base/java.net.URL.openStream(URL.java:1140)
	at DownloadJDK.browse(DownloadJDK.java:42)
	at DownloadJDK.main(DownloadJDK.java:20)
Error: Process completed with exit code 1.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -23,7 +23,7 @@ jobs:
 
       - name: 'Download JDK'
         id: download-jdk
-        uses: sormuras/download-jdk@main
+        uses: ./
         with:
           feature: ${{ matrix.feature }}
 

--- a/java/DownloadJDK.java
+++ b/java/DownloadJDK.java
@@ -17,7 +17,7 @@ class DownloadJDK {
     System.out.println("Download JDK " + feature + " (" + os + "-" + arch + ")");
     // System.getProperties().forEach((k, v) -> System.out.println(k + ":" + v));
 
-    var properties = browse("https://github.com/sormuras/bach/raw/HEAD/install-jdk.properties");
+    var properties = browse("https://github.com/sormuras/bach/raw/master/install-jdk.properties");
     var pro = new Properties();
     pro.load(new StringReader(properties));
     var uri = pro.getProperty(feature + "-" + os + "-" + arch);

